src/Verilator.cpp

```diff
@@ -526,48 +526,8 @@ static void process() {
     // Note early return above when opt.cdc()
 }
 
-//######################################################################
-
-int main(int argc, char** argv, char** env) {
-    // General initialization
-    std::ios::sync_with_stdio();
-
-    time_t randseed;
-    time(&randseed);
-    srand(static_cast<int>(randseed));
-
-    // Post-constructor initialization of netlists
-    v3Global.boot();
-
-    // Preprocessor
-    // Before command parsing so we can handle -Ds on command line.
-    V3PreShell::boot(env);
-
-    // Command option parsing
-    v3Global.opt.bin(argv[0]);
-    string argString = V3Options::argString(argc-1, argv+1);
-    v3Global.opt.parseOpts(new FileLine(FileLine::commandLineFilename()),
-                           argc-1, argv+1);
-
-    // Validate settings (aka Boost.Program_options)
-    v3Global.opt.notify();
-
-    V3Error::abortIfErrors();
-
-    // Can we skip doing everything if times are ok?
-    V3File::addSrcDepend(v3Global.opt.bin());
-    if (v3Global.opt.skipIdentical().isTrue()
-        && V3File::checkTimes(v3Global.opt.makeDir() + "/" + v3Global.opt.prefix()
-                              + "__verFiles.dat", argString)) {
-        UINFO(1,"--skip-identical: No change to any source files, exiting\n");
-        exit(0);
-    }
-    // Undocumented debugging - cannot be a switch as then command line
-    // would mismatch forcing non-identicalness when we set it
-    if (!V3Os::getenvStr("VERILATOR_DEBUG_SKIP_IDENTICAL", "").empty()) {
-        v3fatalSrc("VERILATOR_DEBUG_SKIP_IDENTICAL w/ --skip-identical: Changes found\n");
-    }
-
+static void execVerilate(const string& argString) {
```

Comment:
```suggestion
static void verilate(const string& argsString) {
```

Reply:
Let me clarify a bit.
Do you prefer to rename all argString to argsString in Verilator.cpp, or just in the function verilate() ?
Should V3Options::argString() be renamed to V3Options::argsString() ?

Reply:
You can keep it argString.

Reply:
I see.
renamed in 34b72f359f6e036ca8124b46af26a7afa404406e